config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :greetings
```

Comment:
- [ ] I was unable to make any calls due to blocking from CORS policy. Kindly fix that. In the `config/initializers/cors.rb` file uncomment this line and change "example.com" to "*" to allow requests.

  ```
  Rails.application.config.middleware.insert_before 0, Rack::Cors do
    allow do
      origins "*"
  
      resource "*",
        headers: :any,
        methods: [:get, :post, :put, :patch, :delete, :options, :head]
    end
  end
  ```

  ![wndh](https://user-images.githubusercontent.com/36542986/199689619-350fb657-2068-4648-8318-32c227640894.jpg)

- [ ] Now there are issues on your Front end that are highlighted. Kindly look at the requested changes there.